src/schema/v2/artwork/artworkVisibility.ts

```diff
@@ -0,0 +1,11 @@
+import { GraphQLEnumType } from "graphql"
+
+const artworkVisibility = new GraphQLEnumType({
+  name: "Visibility",
+  values: {
+    UNLISTED: { value: "unlisted" },
+    LISTED: { value: "listed" },
+  },
+})
+
+export default artworkVisibility
```

Comment:
We should always avoid default exports in JS, because that means that devs can import and name whatever they want (which leads to naming drift at scale):

Ie, 

```
export default artworkVisibility

// then later 
import Foo from './artworkVisibility' // oops! we're now 1:2 
```

With named exports:
```tsx
export const artworkVisibility = ...
```
you lock things down at the compiler level:
```
import { Foo } from './artworkVisibility' // error 
import { artworkVisibility } from './artworkVisibility' // good 
```

Reply:
Thanks for the explanation! and adjusted